app/controllers/alaveteli_pro/pricing_controller.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+# Does not inherit from AlaveteliPro::BaseController as is pre-login
+class AlaveteliPro::PricingController < ApplicationController
+  before_filter :check_pro_pricing_enabled
+
+  def index
+  end
+
+  private
+
+  def check_pro_pricing_enabled
+    raise ActiveRecord::RecordNotFound unless feature_enabled?(:pro_pricing)
```

Comment:
It would be better to do this with a [feature constraint in `config/routes.rb`](https://github.com/mysociety/alaveteli/blob/develop/config/routes.rb#L608) – wasn't aware of this until today.